### PR TITLE
Correct used logging levels

### PR DESCRIPTION
**Pull request by leszekgruchala**

### CODE REVIEW

File: kensu/pyspark/spark_connector.py [hunk 1]
```diff
@@ -488,7 +488,7 @@ def init_kensu_spark(
         try:
             config.read(conf_path)
         except:
-            logging.warning(f"Cannot load config from file `%s`" % (conf_path))
+            logging.error(f"Cannot load config from file `%s`" % (conf_path))
```
Comment (vidma) [inline kensu/pyspark/spark_connector.py:491]:
```suggestion
            logging.error(f"Cannot load Kensu config from file `%s`" % (conf_path))
```

File: kensu/pyspark/spark_connector.py [hunk 2]
```diff
@@ -254,10 +254,10 @@ def report_as_kpi(self,  # type: DataFrame
             jvm = get_jvm_from_df(self)
             w = jvm.io.kensu.sparkcollector.KensuSparkCollector.KensuSparkDataFrame(self._jdf)
             w.reportAsKPI(name, logical_name or name, datasource_type)
-            logging.error('KENSU: df.report_as_kpi started in background')
+            logging.info('KENSU: df.report_as_kpi started in background')
         except:
             import traceback
-            logging.info("KENSU: unexpected issue: {}".format(traceback.format_exc()))
+            logging.error("KENSU: unexpected issue: {}".format(traceback.format_exc()))
```
Comment (vidma) [inline kensu/pyspark/spark_connector.py:260]:
```suggestion
            logging.warning("KENSU: unexpected issue: {}".format(traceback.format_exc()))
```

File: kensu/pyspark/spark_connector.py [hunk 3]
```diff
@@ -132,7 +132,7 @@ def wrapper(self):
             logging.info('KENSU: in patched DataFrame.toPandas(), returning result')
         except:
             import traceback
-            logging.info("KENSU: unexpected issue in DataFrame.toPandas(): {}".format(traceback.format_exc()))
+            logging.error("KENSU: unexpected issue in DataFrame.toPandas(): {}".format(traceback.format_exc()))
```
Comment (vidma) [inline kensu/pyspark/spark_connector.py:135]:
```suggestion
            logging.warning("KENSU: unexpected issue in DataFrame.toPandas(): {}".format(traceback.format_exc()))
```

File: kensu/pyspark/spark_connector.py [hunk 4]
```diff
@@ -116,7 +116,7 @@ def wrapper(self):
             logging.info('KENSU: in patched DataFrame.write, returning result')
         except:
             import traceback
-            logging.info("KENSU: unexpected issue in DataFrame.write: {}".format(traceback.format_exc()))
+            logging.error("KENSU: unexpected issue in DataFrame.write: {}".format(traceback.format_exc()))
```
Comment (vidma) [inline kensu/pyspark/spark_connector.py:119]:
```suggestion
            logging.warning("KENSU: unexpected issue in DataFrame.write: {}".format(traceback.format_exc()))
```

File: kensu/pyspark/spark_connector.py [hunk 5]
```diff
@@ -98,7 +98,7 @@ def wrapper(self, data, *args, **kwargs):
             logging.info('KENSU: done modifying arguments to spark.createDataFrame')
         except:
             import traceback
-            logging.info("KENSU: unexpected issue: {}".format(traceback.format_exc()))
+            logging.error("KENSU: unexpected issue: {}".format(traceback.format_exc()))
```
Comment (vidma) [inline kensu/pyspark/spark_connector.py:101]:
```suggestion
            logging.warning("KENSU: unexpected issue: {}".format(traceback.format_exc()))
```

File: kensu/pyspark/spark_connector.py [hunk 6]
```diff
@@ -37,7 +37,7 @@ def wrapper(self):
             logging.info('KENSU: in spark.stop, waiting for Kensu Spark-Collector done, stopping spark context')
         except:
             import traceback
-            logging.info("KENSU: unexpected issue: {}".format(traceback.format_exc()))
+            logging.error("KENSU: unexpected issue: {}".format(traceback.format_exc()))
```
Comment (vidma) [inline kensu/pyspark/spark_connector.py:40]:
```suggestion
            logging.warning("KENSU: unexpected issue: {}".format(traceback.format_exc()))
```

**Review by vidma — COMMENTED** [review]
@leszekgruchala this is kinda particular thingy depending on point of view:
- from Kensu collector: it is error
- **however, from point of view of customer's app, it's just a WARNING at max** if it does not break the main app logic. The customer app normally should not crash and not be affected by our collectors (unless requested explicitly by some config option), and in case they were monitoring they're logs with some tool, we usually want to avoid ERROR messages in logs.

however, good you noticed it, as we might useful to double check if we are not sometimes swallowing any of the original APP errors in the `except` clause of some of our wrappers.

**Comment by vidma** [comment]
this PR might be too boring for @lelayf , but still adding as discussion might be interesting.

**Comment by leszekgruchala** [comment]
Got the point. Still, for all those "unexpected issues" we should report an ERROR. @SammyEK do you think for other cases a WARN is enough?

**Comment by vidma** [comment]
> Got the point. Still, for all those "unexpected issues" we should report an ERROR. @SammyEK do you think for other cases a WARN is enough?

well, imho no - at least in spark collector we were very clear on this so far, no matter what the issue, if it doesn't break customer's app/code/job  it should be not ERROR...(but we should review kensu-py on this, it's not following this in many places!)

File: kensu/pyspark/spark_connector.py (hunk 1 above)
Comment (vidma) [reply kensu/pyspark/spark_connector.py:491]: @leszekgruchala this is the only exception i agree with, conf file changes rarely, and is not impacted by us directly, so useful to catch/notice conf issues

**Comment by SammyEK** [comment]
Indeed Kensu's philosophy is to avoid breaking the customer code, so for Kensu related events I'd use `warn` level instead of error

**Comment by vidma** [comment]
@SammyEK don't we need some a new flag in kensu-py which would not break customer code in case of some kensu issues (don't remember exactly but something like: misconfig, something not connecting, error in our wrappers)? or should we leave it until we do some big/huge refactoring?

**Comment by SammyEK** [comment]
that would be great, do you know how long it should take to implement this? (Today the priority is not on Kensu-py, but well on pyspark)

**Review by vidma — APPROVED** [review]

**Pull request merged**